app/representers/api/v1/unclaimed_user_representer.rb

```diff
@@ -0,0 +1,13 @@
+module Api::V1
+  class UnclaimedUserRepresenter < Roar::Decorator
```

Comment:
I see your point about the user not explicitly authorizing the application, but don't we have to assume they're trusting the application?  We're allowing the application to create data for them before they've even signed up, and then transferring that data to their account once they confirm their email address.

If that's an issue, perhaps we need to re-think how the claiming process works and present a message to the newly signed up user saying something like "Application Foo has entered some information on your behalf, do you want to authorize Foo to do so?"

But then users may click "no" out of reflex, not understanding that they'll lose important data by doing so.